components/team_card/team_card_storage.lua

```diff
@@ -29,6 +33,7 @@ function TeamCardStorage.saveToLpdb(args, teamObject, players, playerPrize)
 			team = teamObject.lpdb
 		end
 	end
+	---@cast team -nil
```

Comment:
should we do an assert here?
code further down needs team to not be nil while the above code doesn't insure that